ui/src/app/components/workflows/workflow/jobs/job/job.component.ts

```diff
@@ -59,7 +59,7 @@ export class JobComponent implements OnInit, OnDestroy {
       this.selectedJob = !!selected ? selected.value : undefined;
     });
 
-    this.jobChangesSubscription = this.jobChanges.subscribe((jobChange) => {
+    this.jobChangesSubscription = this.jobChanges.pipe(delay(0)).subscribe((jobChange) => {
```

Comment:
Why is that needed? delay of zero seems redundant..

Reply:
Yes you are right. Removed

Reply:
> ![image](https://user-images.githubusercontent.com/54136196/83737171-ef449600-a652-11ea-8cec-6bc3defe1f23.png)
> Two problems here:
> 
> 1. Wrong error message
> 2. Maxlength too small
> 
> Same for main class and value za.co.absa.hyperdrive.driver.drivers.CommandLineIngestionDriver

Fixed